src/str/adapters.rs

```diff
@@ -1,9 +1,9 @@
 use std::ffi;
 use std::fmt;
 use std::str;
-
 use Predicate;
```

Comment:
Why was this newline removed?  I prefer a style where local `use` are kept separate from `std` use